src/Generators/Google.php

```diff
@@ -21,19 +21,9 @@ public function generate(Link $link): string
     {
         $url = 'https://calendar.google.com/calendar/render?action=TEMPLATE';
 
-        $utcStartDateTime = (clone $link->from)->setTimezone(new DateTimeZone('UTC'));
-        $utcEndDateTime = (clone $link->to)->setTimezone(new DateTimeZone('UTC'));
         $dateTimeFormat = $link->allDay ? $this->dateFormat : $this->dateTimeFormat;
-        $url .= '&dates='.$utcStartDateTime->format($dateTimeFormat).'/'.$utcEndDateTime->format($dateTimeFormat);
-
-        // Add timezone name if it is specified in both from and to dates and is the same for both
-        if (
-            $link->from->getTimezone() && $link->to->getTimezone()
-            && $link->from->getTimezone()->getName() === $link->to->getTimezone()->getName()
-        ) {
-            $url .= '&ctz=' . $link->from->getTimezone()->getName();
-        }
-
+        $url .= '&dates='.$link->from->format($dateTimeFormat).'/'.$link->to->format($dateTimeFormat);
+        $url .= '&ctz=UTC';
```

Comment:
This formats dates in UTC (using `Ymd\THis\Z`) like the other generators do, and forces `ctz=UTC`